app/views/posts/index.html.erb

```diff
@@ -1,2 +1,37 @@
-<h1>Load all post of the user</h1>
+<section class="show_user_posts d-flex flex-column">
+    <div class="user-lists ">
+        <%= render "user_header", :user =>@user %>
+    </div>
+
+    <div class=posts>
+    <% (1..5).each do |i|%>
+         <div class="post">
+             <%= link_to user_post_path(@user.id, i),  class: "post-p" do %>
+            
+                <div class="post-border post-p">
+                    <h3>Post#<%= i %> </h3>
+                    <p> 
+                        lorem ipsum dolor sit amet, consectetur adipisicing elit. lorem*ipsum dolor sit amet, consectetur adipisicing elit.
+                    </p>
+                    <div class="comment-num-2">Comments: 10, Likes: 10k</div>
```

Comment:
- [ ] Same to the `comments` and `likes` you can't add them here as just a simple div you can create them in the `rails console`, I mean if you have `users` and `posts` you can give a `comment` or a `like` and they should be incremented automatically because you have methods in your models that do it go back and check.